Plugin/DotpayDirectPlugin.php

```diff
@@ -503,11 +503,7 @@ protected function getReturnUrl(ExtendedDataInterface $data)
     private function getCallBackUrl(PaymentInstructionInterface $instruction, $route)
     {
         if ($this->callBackUrl) {
-            return $this->callBackUrl.$this->router->generate(
-                $route,
-                ['id' => $instruction->getId()],
-                Router::RELATIVE_PATH
-            );
+            $this->router->getContext()->setBaseUrl($this->callBackUrl);
```

Comment:
- would be great to check that it's possible to do that with all symfony versions supported by this bundle (it seems it's ok for version 5 at least: this method is available in https://github.com/symfony/routing/blob/5.x/RequestContextAwareInterface.php#L26 implemented by the `Router` + https://github.com/symfony/routing/blob/5.x/RequestContext.php#L99)
- would have been great to unit test this case.